pkg/agent/controller/noderoute/node_route_controller_unix.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020 Antrea Authors
```

Comment:
Since netlink works for linux only and you are limiting it via build tag, it should be named _linux.go directly, otherwise it may give the impression that it works for other unix-like OSes. Then you don't have to add the build tag as the filename is an implicit `+build linux` build tag (but I'm not against keeping it explicitly). 
BTW, I might be not clear, I was actually asking to remove the two chars "//" between the comments and the build tag but still keep the empty line to **separate** them clearly.